scripts/reset-test-db.sh

```diff
@@ -92,7 +92,7 @@ else
 
         # Dump the schema and data to the cache
         echo "Using the dumpdata command to save the $db fixture data to the filesystem."
-        ./manage.py lms --settings bok_choy dumpdata --database $db > $DB_CACHE_DIR/bok_choy_data_$db.json
+        ./manage.py lms --settings bok_choy dumpdata --exclude api_admin.Catalog --database $db > $DB_CACHE_DIR/bok_choy_data_$db.json
```

Comment:
This line is the change from https://github.com/edx/edx-platform/pull/12395 required to keep this script working, I likely won't be including it in this PR's merge.